### ISSUE TITLE
pre-trained model

### ISSUE THREAD

**Issue by asdfqwer2015** [ahmetgunduz/Real-time-GesRec#2]
Hi, would you like to share your pre-trained model that can be finetuned for both detection and classification?
Thanks.

**Comment by asdfqwer2015** [comment]
En, I noticed "We thank Kensho Hara for releasing 3D-ResNets-PyTorch Repo, which we build our work on top." in readme. Should the model be pretrained with kinetics before finetuning with nvgesture/egogesture or be trained with nvgesture/egogesture from scratch?
Thanks.

**Comment by ahmetgunduz** [comment]
Hi. I used Kensho Hara's repo as a basis but I have my own models I wanted to share. I will add download link to the models in a week. Sorry that I have been busy for some time.

**Comment by brian-yu** [comment]
Hi, there. This paper is really amazing! Are there pretrained models available for the Jester dataset? Thanks!

**Comment by ahmetgunduz** [comment]
Thanks Brian, There was but I need to check if I still have them. In case you want to train a model on Jester, you can follow the instructions on README,

**Comment by ahmetgunduz** [comment]
Hi @asdfqwer2015 and @brian-yu , Please see the updated README file for the google drive link to pretrained models.

I am closing the issue.

**Comment by asdfqwer2015** [comment]
That's great. Thanks.
:)

**Comment by ahmetgunduz** [comment]
enjoy!! :)

**Comment by Shaotran** [comment]
> Hi @asdfqwer2015 and @brian-yu , Please see the updated README file for the google drive link to pretrained models.
> 
> I am closing the issue.

Hi there,

I am trying to use the pretrained ResNeXT Jester model for recognizing hand gestures. When I try using the model, it's practically guessing -- am I to assume that the .pth has random weights and we need to train the model to recognize Jester gestures ourselves?

Thanks!

**Comment by ahmetgunduz** [comment]
@Shaotran Can you please share your parameter in bash file?

**Comment by Shaotran** [comment]
> @Shaotran Can you please share your parameter in bash file?

I've attached my code below for the ResNeXT-101 EgoGesture model/dataset. I've noticed 2 issues: (a) The model seems to predict a seemingly random output. (b) Every time the model is re-loaded from the .pth, its prediction changes to something else. Any ideas?

I am using Python 3.6.8. All the settings (num_classes, sample_size, norm_value, etc.) are all set in the code; perhaps I set one of those to the wrong value?

[Gestures (1).py.zip](https://github.com/ahmetgunduz/Real-time-GesRec/files/3408036/Gestures.1.py.zip)
